.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+# Jira Ticket: 
+
+# Description of Work
```

Comment:
One `#` is H1, which is loud; I would suggest using `##` for something essential and `###` for something you want but isn't the most critical information for the reader.

Typically `#` is reserved for the title of a wiki or README.

Reply:
That's good to know thanks, usually I just begin using H1 for heading H2 for subheadings, etc. I will update these to use ## and ###